BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
+- identifier: RECOMMENDED
+  name: REQUIRED
+  url: RECOMMENDED
```

Comment:
URL to NIMH in general, or to the grant page in the NIH reporter? Example:
https://projectreporter.nih.gov/project_info_description.cfm?aid=9584926&icde=42044474

Reply:
to reporter page.